templates/header-left.mustache

```diff
@@ -1,6 +1,6 @@
 <div class="brand">
 	<div class="logo">
-		<a href="{{urlTo 'canjs'}}" alt="CanJS"></a>
+		<a href="{{urlTo 'canjs'}}" alt="CanJS" tabindex="-1"></a>
```

Comment:
I don’t think this is a good idea because a user might want to go to the home page.